src/driftpy/priority_fees/priority_fee_subscriber.py

```diff
@@ -18,7 +18,7 @@ class PriorityFeeConfig:
 class PriorityFeeSubscriber:
     def __init__(self, config: PriorityFeeConfig):
         self.connection = config.connection
-        self.frequency_ms = config.frequency_secs
+        self.frequency_secs = config.frequency_secs
```

Comment:
i'd keep this ms to avoid breaking change. marginal convenience added by converting to seconds from ms